Automattic/wp-calypso
# Components: Use `KeyringServices` rather than `lib/services-list`

Moves away from `lib/services-list` to using `KeyringServices` built on Redux.

To test:
1. Go to `My Sites`.
1. Click `Sharing`.

Two lists of services should load, “Publicize Your Posts” and “Other Connections”. They should display a button inline with the correct action, most likely “Connect”.

See #8553.
Fixes #7919.
